docs/user_manual/source/control_status_registers.rst

```diff
@@ -1031,14 +1031,14 @@ Machine Architecture ID (``marchid``)
 
 CSR Address: 0xF12
 
-Reset Value: 0x0000_0004
+Reset Value: 0x0000_0015
 
 Detailed:
 
 +-------------+-----------+------------------------------------------------------------------------+
 |   Bit #     |   Mode    |   Description                                                          |
 +=============+===========+========================================================================+
-| 31:0        | RO        | Machine Architecture ID of |corev| is 4                                |
+| 31:0        | RO        | Machine Architecture ID of |corev| is 0x15                             |
```

Comment:
I would set this number in degimal format as the RISC-V official repository and the CV32E40P manual. Only the one in the description. The reset value should stay in hex format

Reply:
Hi @davideschiavone It seems that you are looking at an old version of that file. The most recent commit includes the digital representation as well.

Reply:
you are right sorry. pls go ahead with merge